app/modules/sightings/resources.py

```diff
@@ -428,7 +455,29 @@ def patch(self, args, sighting):
             )
 
         if edm_count > 0:
+            cleanup = SightingCleanup()
             log.debug(f'wanting to do edm patch on args={args}')
+
+            # we pre-check any annotations we will want to attach to new encounters
+            enc_anns = (
+                []
+            )  # list of lists of annotations for each encounter (if applicable)
+            try:
+                for arg in args:
```

Comment:
With the suggested change to _get_annotations this would need to be passed as a list which I think could be done using list comprehension:
enc_list = [arg.get('value', {}) for arg in args if arg.get( 'path', None) == '/encounters' and arg.get('op', None) == 'add']
which of course brunette will smear across 10 lines :)